### ISSUE TITLE
Pressing button with href tag reveals teal tint

### ISSUE THREAD

**Issue by JeffreyCA** [TerryMooreII/materialize-social#3]
I have a button defined as follows:

```
<a class="waves-effect waves-light btn-floating social google" data-delay="100" href="mailto:example@example.com">
 <i class="fa fa-google"></i></a>
```

After clicking the button, the background colour of the button becomes a teal/green colour (see screenshot).

<img width="60" alt="screenshot 2017-09-04 18 10 40" src="https://user-images.githubusercontent.com/9157833/30039281-d736ad96-919c-11e7-8be1-9d1214889c3f.png">

# Solution
Adding `!important` annotations besides the `background-color` in the .css file fixes this issue.
Example:

```css
.linkedin {
  background-color: #007bb6 !important;
  color: #fff !important;
}
```

There is no teal tint if there is no href tag in the button.

**Comment by TerryMooreII** [comment]
Ok, I will take a look and get that updated.

**Comment by JeffreyCA** [comment]
Thanks

**Comment by TerryMooreII** [comment]
Added the !important.  Published in version 1.0.2
